hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -230,6 +230,37 @@ def hpat_pandas_series_index_impl(self):
     return hpat_pandas_series_index_impl
 
 
+@overload_attribute(SeriesType, 'size')
+def hpat_pandas_series_size(self):
+    """
+    Pandas Series attribute :attr:`pandas.Series.size` implementation
+
+    **Algorithm**: result = series.size
```

Comment:
No need this section. Please refer to other methods

Reply:
Sorry, I do not understand. I see lines like this in many other methods.
I will remove `Algorithm` and add `.. only:: developer`. Is it ok?

Reply:
ok, never mind, leave it as is.